src/redis-default.js

```diff
@@ -0,0 +1,104 @@
+import { promisify } from 'util'
+
+class RedisDefaultStore {
+  constructor (client, options = {}) {
+    const invalidClientError = new TypeError(
+      'First parameter must be a valid RedisClient instance.'
+    )
+
+    try {
```

Comment:
this try catch seems redundant.

Reply:
This try-catch might be if `client.constructor` is undefined